ftp-test-utils/src/main/java/org/ballerinalang/stdlib/ftp/testutils/mockServerUtils/MockFTPServer.java

```diff
@@ -102,11 +138,179 @@ public static void initServer(BMap<Object, Object> config) throws BallerinaFTPEx
         logger.info("Started Mock FTP server");
     }
 
-    public static void stopServer() {
+    public static void startFtpsServer(BMap<Object, Object> config) throws FtpException, BallerinaFTPException {
+                final FtpServerFactory serverFactory = new FtpServerFactory();
+        int port = FTPUtil.extractPortValue(config.getIntValue(StringUtils.fromString(
+                FTPConstants.ENDPOINT_CONFIG_PORT)));
+        final BMap auth = config.getMapValue(StringUtils.fromString(
+                FTPConstants.ENDPOINT_CONFIG_AUTH));
```

Comment:
```suggestion
        final BMap auth = config.getMapValue(StringUtils.fromString(FTPConstants.ENDPOINT_CONFIG_AUTH));
```

Reply:
Could not get the difference.

Reply:
I just made the 2 lines into a single line. But that's fine.